src/app/components/AllowanceMenu/index.tsx

```diff
@@ -145,26 +208,65 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
                 onChange={(e) => setBudget(e.target.value)}
               />
             </div>
-            <Setting
-              title={t("enable_login.title")}
-              subtitle={t("enable_login.subtitle")}
+            <div
+              className={
+                hasPermissions
+                  ? "pb-4 border-b border-gray-200 dark:border-neutral-500"
+                  : ""
+              }
             >
-              <Toggle
-                checked={lnurlAuth}
-                onChange={() => setLnurlAuth(!lnurlAuth)}
-              />
-            </Setting>
+              <Setting
+                title={t("enable_login.title")}
+                subtitle={t("enable_login.subtitle")}
+              >
+                <Toggle
+                  checked={lnurlAuth}
+                  onChange={() => setLnurlAuth(!lnurlAuth)}
+                />
+              </Setting>
+            </div>
+
+            {hasPermissions && (
+              <>
+                <h2 className="mt-4 text-lg text-gray-900 font-bold dark:text-white">
+                  {t("edit_permissions")}
+                </h2>
+                <div>
+                  {permissions.map((permission, index) => (
+                    <div key={index} className="flex items-center my-2">
+                      <Checkbox
```

Comment:
You still wanted to switch this to be a toggle?

Reply:
We're deleting them now, right? Wdyt? Should we change this?

Reply:
Hm, don't think what happens inside is important or needs to be reflected in the UI.  
Eventually we might use enable/disable anyways, right?  
Also we currently do not use checkboxes in this kind of situation.

Reply:
Makes sense. Toggle it is then!